ci/travis/release-py3x.yml

```diff
@@ -0,0 +1,4 @@
+numpy: 1.23.5
+python: 3-builtin
+setuptools: 65.6.3
+wheel: 0.38.4
```

Comment:
Could bumping more recent versions have any unexpected side effect?

Reply:
I don't think so, I think they just serve as an example to test the komodo scripts, that this release matches with the repository etc. But they had to be bumped because it actually pulls these off pypi, and the versions pulled must be supported on the Python version we test for.